packages/lit-dev-content/samples/tutorials/CONTRIBUTING.md

```diff
@@ -189,36 +189,36 @@ customElements.define('my-element', MyElement);
 You can also insert an aside in your instructions by using the following format:
 
 ```html
-<litdev-aside type="positive">
+{% aside "positive" %}
 
 The first line is always a bolded header.
 
 Make sure to do `this`!
 
-</litdev-aside>
+{% endaside %}
 
-<litdev-aside type="warn" no-header>
+{% aside "warn" no-header %}
```

Comment:
Does this need wrapping quotes around `"no-header"`?

Reply:
si